packages/radix-vue/src/shared/component/Arrow.vue

```diff
@@ -38,7 +38,7 @@ const { primitiveElement } = usePrimitiveElement()
     v-bind="props"
     :width="width"
     :height="height"
-    viewBox="0 0 30 10"
+    :viewBox="asChild ? undefined : '0 0 30 10'"
     preserveAspectRatio="none"
```

Comment:
Icic! Let's do the same for `preserveAspectRatio` too. I imagine there's a case where dev might want to change this.

Reply:
> Let's do the same for preserveAspectRatio too. I imagine there's a case where dev might want to change this.

I missed. We should do that!👍